entity-server-api/pom.xml

```diff
@@ -40,5 +40,10 @@
       <artifactId>entity-common-api</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
It does seem like it might be better to create an entirely new exception, rather than stretch EntityUserException to fit.